content/docs/resources/standard-routes.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Standard Routes"
+weight: 7
+---
+
+### Cases
+
+{{< route method="POST" route="/gossip/v1/cases" >}}
+
+This endpoint notifies the server of a new case
+
+Request Body:
+
+| Field      | Type.       | Description                                                          |
+| ---------- | ----------- | -------------------------------------------------------------------- |
+| `guild`    | snowflake   | Guild ID where the case took place.                                  |
+| `user`     | snowflake   | User ID where the action was taken.                                  |
+| `actioner` | snowflake   | User ID that created the case. Might be bot ID if it was automated.  |
+| `action`   | case_action | Type of action taken.                                                |
+| `duration` | long.       | Milliseconds for duration. 0 if permanent                            |
+| `reason`   | string.     | Reason for action, might be empty                                    |
+
+
+Where `case_action` is one of (`BAN`, `KICK`, `MUTE`, `WARN`)
+
+Example Body:
+
+```json
+{
+  "data":{
+    "type":"member_join",
```

Comment:
I don't understand what the `type` parameter here does?

Reply:
uh, bad copy paste haha